### DISCUSSION TITLE
Alternative to DUALSHOCK

### DISCUSSION THREAD

**Discussion by yoshida6298** [HumanSupportRobot/QandA#103] [Q&A]
Both PS3 and PS4 controllers have become difficult to obtain recently. Do you know of any good alternatives?

**Comment by fabiendespinoy** [comment]
For some robots, we are using a Gioteck VX3 Wireless
I used it yesterday with HSR(C), so I confirm it works.

**Comment by alex-mitrevski** [comment]
We use a Logitech controller with our HSR, with a custom-made node to map key presses to desired commands. That has always worked reliably for us.
